pkg/pillar/cmd/zedrouter/server.go

```diff
@@ -336,3 +344,25 @@ func (hdl openstackHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 	w.WriteHeader(http.StatusNotFound)
 }
+
+// ServeHTTP for kubeConfigHandler provides cluster kube config
+func (hdl kubeConfigHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	var kubeConfig types.KubeConfig
```

Comment:
Isn't there a number of checks missing here such as checking it is a POST (and returning an error if not), checking that the content-type is json (and return error), and also check against some maximum size (to avoid OOM if an app sends a Gbyte of data).

Reply:
Resolved.